### PR TITLE
cov: add error checking tests for distributed/launcher.py 

### PR DESCRIPTION
**Pull request by ydcjeff**
Part of #1790 

Description: Add error checking tests for launcher.py

Check list:

- [x] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by sdesrozis** [comment]
@ydcjeff Thank you ! Why not add wrong inputs for the static method `_setup_spawn_params` ?

File: tests/ignite/distributed/test_launcher.py [hunk 1]
```diff
@@ -10,6 +10,29 @@
 from ignite.distributed.utils import has_hvd_support, has_native_dist_support, has_xla_support
 
 
+def test_parallel_error():
```
Comment (sdesrozis) [inline tests/ignite/distributed/test_launcher.py:13]:
```suggestion
def test_parallel_wrong_inputs():
```

**Comment by ydcjeff** [comment]
> @ydcjeff Thank you ! Why not add wrong inputs for the static method `_setup_spawn_params` ?

Since its private method, I go from `Parallel` also going from `Parallel` is readable.

**Comment by sdesrozis** [comment]
> Since its private method, I go from `Parallel` also going from `Parallel` is readable.

My bad ! You are totally right, this method is covered by the tests.

**Pull request merged**